### PR TITLE
Adding outputDir to standalone

### PR DESCRIPTION
**Pull request by Gilad-WT**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)
Added outputDir option for standalone (no testrunner) 
## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [X] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by Gilad-WT** [comment]
@christian-bromann Added the feature we talked about here: https://github.com/webdriverio/webdriverio/issues/3580

**Comment by Gilad-WT** [comment]
@christian-bromann Added test

**Comment by Gilad-WT** [comment]
> I still don't understand how this fixes your problem. The `WDIO_LOG_PATH` environment variable is being used in the testrunner but you said you don't use the testrunner.

@christian-bromann From what I've seen in the code WDIO_LOG_PATH is used by wdio-reporter which is the one doing the reporting for both wdio and by the testrunner.

**Comment by Gilad-WT** [comment]
@christian-bromann I also validated it with this code in a new dir:
```const wdio = require('webdriverio')

async function test(){
    let browser = await wdio.remote({ 
        host: 'localhost', 
        outputDir: "./",
        port: 4444,
        logLevel : "info",
        capabilities:{ 
            browserName: 'chrome',
        }
    });
    return browser
}
test()
```

**Comment by christian-bromann** [comment]
> reporting for both wdio and by the testrunner.

If you run WebdriverIO in standalone mode you won't use these packages.

**Comment by Gilad-WT** [comment]
@christian-bromann it is imported by middlewares.js

**Comment by christian-bromann** [comment]
> it is imported by middlewares.js

I don't understand

**Comment by Gilad-WT** [comment]
@christian-bromann From what I have seen in the code looger is used to log by both wedriverio and webdriver and it is set by this line:
import logger from '@wdio/logger' (line 1 of webdriver/index.js).
So it looks like wd and wdio use wdio/logger to log

File: packages/webdriverio/tests/module.test.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ jest.mock('@wdio/config', () => {
 })
 
 const WebDriver = require('webdriver')
+import path from 'path'
```
Comment (christian-bromann) [inline packages/webdriverio/tests/module.test.js]: Let's move this import to the top of the file.

File: packages/webdriverio/src/index.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { validateConfig, wrapCommand, runFnInFiberContext, detectBackend } from
 import MultiRemote from './multiremote'
 import { WDIO_DEFAULTS } from './constants'
 import { getPrototype } from './utils'
+import path from 'path'
```
Comment (christian-bromann) [inline packages/webdriverio/src/index.js]: Also this import needs to move up to the first line

Comment (Gilad-WT) [reply packages/webdriverio/src/index.js]: fixed

File: packages/webdriverio/src/index.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ export const remote = async function (params = {}, remoteModifier) {
     instance.addCommand = (name, fn, attachToElement) => (
         origAddCommand(name, runFnInFiberContext(fn), attachToElement)
     )
-
+
```
Comment (christian-bromann) [inline packages/webdriverio/src/index.js]:
```suggestion
```

Comment (Gilad-WT) [reply packages/webdriverio/src/index.js]: @christian-bromann I thought you meant to move it in the code but I now understand you mean in the docs. I'll change it back and fix the docs tomorrow.

**Comment by Gilad-WT** [comment]
@christian-bromann I made the changes you requested please take a look

**Comment by Gilad-WT** [comment]
@christian-bromann Thanks

**Pull request merged**